SimPEG/utils/model_utils.py

```diff
@@ -1,9 +1,10 @@
-from .mat_utils import mkvc, ndgrid, uniqueRows
+from .mat_utils import mkvc
 import numpy as np
-from scipy.interpolate import griddata, interp1d
-from scipy.interpolate import NearestNDInterpolator, LinearNDInterpolator, interp1d
+from scipy.interpolate import griddata
 from scipy.spatial import cKDTree
 import scipy.sparse as sp
+from discretize.utils import active_from_xyz
+import warnings
 
 
 def surface2ind_topo(mesh, topo, gridLoc="CC", method="nearest", fill_value=np.nan):
```

Comment:
Make surface2ind_topo return the same thing it did before please. Even though it is deprecated, it's functionality should be unchanged.

Probably just need to do:

```
active_cells = active_from_xyz(mesh, topo, gridLoc, method)
return np.arange(mesh.n_cells)[active_cells]
```